src/account/mod.rs

```diff
@@ -79,9 +81,19 @@ impl<'a> AccountInitialiser<'a> {
             client_options,
             skip_persistance: false,
             storage_path,
+            #[cfg(feature = "stronghold")]
+            account_type: Some(SignerType::Stronghold),
+            #[cfg(not(feature = "stronghold"))]
+            account_type: None,
         }
     }
 
+    /// Sets the account type.
+    pub fn account_type(mut self, account_type: SignerType) -> Self {
```

Comment:
Seems weird to me that account_type is of type SignerType

Reply:
changed it to be a little more consistent.